lib/client.js

```diff
@@ -1,4 +1,5 @@
 const axios = require("axios");
+const { version } = require("uuid");
```

Comment:
![image](https://github.com/peviitor-ro/peviitor_jsscraper/assets/55687464/d6800716-88cc-40f0-a533-c49bc0bffc50)

Reply:
copilot is to blame for this